controller/network/service.go

```diff
@@ -89,7 +89,18 @@ func (ctrl *ServiceController) newModelEntity() boltEntitySink {
 func (ctrl *ServiceController) terminatorChanged(params ...interface{}) {
 	for _, entity := range params {
 		if terminator, ok := entity.(*db.Terminator); ok {
```

Comment:
Terminator updates were being ignored sometimes because the patch didn't have the service so nothing was cleared from cache